packages/docs/src/pages/components/progress/examples/ExValues.vue

```diff
@@ -63,12 +63,22 @@
     </section>
 </template>
 
-<script>
-    export default {
+<script lang="ts">
+    import { defineComponent } from 'vue'
+    import { BField, BNumberinput, BProgress, BSelect, BSwitch } from '@ntohq/buefy-next'
+
+    export default defineComponent({
+        components: {
+            BField,
+            BNumberinput,
+            BProgress,
+            BSelect,
+            BSwitch
+        },
         data() {
             return {
                 indeterminate: false,
-                type: null,
+                type: "",
```

Comment:
@wesdevpro I think that the intention of the original code using `null` was to indicate the "default" value for the `type` prop. To make the `type` prop fall back to the default value without causing any type error, we have to specify `undefined` here, or make `Progress` explicitly allow `null`. The latter I am not inclined to do, though.

Reply:
An empty string won't fall back to the default value `"is-darkgrey"`.

Reply:
You are correct